### PR TITLE
Direct page just layout

### PR DESCRIPTION
**Pull request by e-garbuzov-wise**
### Summary:
Create direct page layout with mock data 

### Changes:
- set layout UI elements of direct page 
- add SnapKit constraints tool

### CODE REVIEW

**Comment by e-garbuzov-wise** [comment]
https://stackoverflow.com/questions/67185817/package-resolved-file-is-corrupted-or-malformed
вот по этой статье иду

File: Instagram-like-trainee/Coordinators/TabsCoordinators/HomeCoordinator.swift [hunk 1]
```diff
@@ -32,7 +32,6 @@ class HomeCoordinator: HomeCoordinatorProtocol {
 
     func didPressDirect() {
         let directViewController = DirectPageViewController()
-        directViewController.title = "direct"
```
Comment (EugeneKirilovIOSDev) [inline Instagram-like-trainee/Coordinators/TabsCoordinators/HomeCoordinator.swift:35]: Move to constants?

File: Instagram-like-trainee/Screens/DirectPage/View/CollectionViewCell/DirectHeaderViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+//  DirectHeaderViewCell.swift
+//  Instagram-like-trainee
+//
+//  Created by Eldar Garbuzov on 12.04.24.
+//
+
+import UIKit
+import SnapKit
+
+class DirectHeaderViewCell: UICollectionReusableView {
```
Comment (EugeneKirilovIOSDev) [inline Instagram-like-trainee/Screens/DirectPage/View/CollectionViewCell/DirectHeaderViewCell.swift:11]: final?

File: Instagram-like-trainee/Screens/DirectPage/View/CollectionViewCell/DirectNotes/DirectNotesViewCell.swift [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+//  DirectNotesViewCell.swift
+//  Instagram-like-trainee
+//
+//  Created by Eldar Garbuzov on 12.04.24.
+//
+
+import UIKit
+import SnapKit
+
+class DirectNotesViewCell: UICollectionViewCell {
```
Comment (EugeneKirilovIOSDev) [inline Instagram-like-trainee/Screens/DirectPage/View/CollectionViewCell/DirectNotes/DirectNotesViewCell.swift:11]: Final?

File: Instagram-like-trainee/Screens/DirectPage/View/CollectionViewCell/DirectPageCell.swift [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+//  DirectPageCell.swift
+//  Instagram-like-trainee
+//
+//  Created by Eldar Garbuzov on 10.04.24.
+//
+
+import UIKit
+import SnapKit
+
+class DirectPageCell: UICollectionViewCell {
```
Comment (EugeneKirilovIOSDev) [inline Instagram-like-trainee/Screens/DirectPage/View/CollectionViewCell/DirectPageCell.swift:11]: Final?

File: Instagram-like-trainee/Screens/DirectPage/View/CollectionViewCell/DirectPageCell.swift [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+        return label
+    }()
+    private let messagePreviewLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = .systemFont(ofSize: 14, weight: .regular)
+        label.text = R.string.localizable.messagePreviewLabel()
+        return label
+    }()
+    lazy var cameraButton: UIButton = {
```
Comment (EugeneKirilovIOSDev) [inline Instagram-like-trainee/Screens/DirectPage/View/CollectionViewCell/DirectPageCell.swift:44]: private?

File: Instagram-like-trainee/Screens/DirectPage/View/CollectionViewCell/DirectPageCell.swift [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+//
+//  Created by Eldar Garbuzov on 10.04.24.
+//
+
+import UIKit
+import SnapKit
+
+class DirectPageCell: UICollectionViewCell {
+    private let label = UILabel()
+    override init(frame: CGRect) {
```
Comment (EugeneKirilovIOSDev) [inline Instagram-like-trainee/Screens/DirectPage/View/CollectionViewCell/DirectPageCell.swift:13]: Why init is between properties?

File: Instagram-like-trainee/Screens/DirectPage/View/DirectPageViewController.swift [hunk 7]
```diff
@@ -8,17 +8,152 @@
 import UIKit
 
 class DirectPageViewController: UIViewController {
-
```
Comment (EugeneKirilovIOSDev) [inline Instagram-like-trainee/Screens/DirectPage/View/DirectPageViewController.swift:11]: Final?

File: Instagram-like-trainee/Screens/DirectPage/View/DirectPageViewController.swift [hunk 8]
```diff
@@ -8,17 +8,152 @@
...
+    private var collectionView: UICollectionView = {
+        let supItem = NSCollectionLayoutBoundarySupplementaryItem(
```
Comment (EugeneKirilovIOSDev) [inline Instagram-like-trainee/Screens/DirectPage/View/DirectPageViewController.swift:12]: Please follow structure, where first you have private let/var and then more opened

**Review by EugeneKirilovIOSDev — CHANGES_REQUESTED** [review]

**Review by mlchct — APPROVED** [review]

**Review by EugeneKirilovIOSDev — APPROVED** [review]

**Review by EugeneKirilovIOSDev — APPROVED** [review]

**Pull request merged**